sdk/nodejs/helm/v2/helm.ts

```diff
@@ -214,6 +219,18 @@ export class Chart extends yaml.CollectionComponentResource {
     }
 }
 
+// ignoreHelmTestHook returns `null` if a resource has an annotation that signals it's part of the
+// Helm test lifecycle hook. This can be provided to Chart's `transformations` API to have it omit
+// all resources that are part of this hook.
```

Comment:
The last sentence isn't true is it - this is automatically added and is not exposed so users can't do it themselves?

Reply:
That's what I meant. This comment isn't for users, it's for maintainers of `helm.ts`. But I will reword.